src/main/java/org/commcare/formplayer/application/FormSubmissionController.java

```diff
@@ -84,14 +105,17 @@ public class FormSubmissionController extends AbstractBaseController {
 
     private final Log log = LogFactory.getLog(FormSubmissionController.class);
 
+    @Value("${form.submit.max_attachments}")
```

Comment:
@Robert-Costello Requesting one more change here. We should rename this property here to `formplayer.form.submit.max_attachments` to have it namespaced as `formplayer`. I missed out on this feedback from Simon earlier.